manual/opt.tex

```diff
@@ -386,15 +388,14 @@ \subsubsection{quartic}
 \end{lstlisting}
 
 \subsection{General recommendations}
-Here are a few reminders to make wavefunction easier.
+Here are a few recommendations to make wavefunction optimization easier.
 \begin{itemize}
-\item All electron wavefunctions are more difficult to optimize than pseudopotential ones especially when the cusp is not removed.
-\item Two body Jastrow contributes the largest portion of correlation energy from bare Slater determinants. It's also the easiest part for optimizer and gains most. For this reason, the recommend order optimizing wavefunction components is two-body, one-body, three-body Jastrow factors and MSD coefficients.
-\item Never use all-zero two-body spline Jastrow and always start from a reasonable one. 
-      You might want to bang your head against a brick wall for a bad two-body Jastrow.
-\item One-body spline Jastrow from old calculations can be good starting point.
-\item Three-body polynomial Jastrow should start from zero.
+\item All electron wavefunctions are typically more difficult to optimize than pseudopotential ones due to the importance of the wavefunction near the nucleus.
+\item Two body Jastrow contributes the largest portion of correlation energy from bare Slater determinants. It's also the easiest part for optimizer and gains most. For this reason, the recommend order optimizing wavefunction components is two-body, two-body, three-body Jastrow factors and MSD coefficients.
```

Comment:
easiest part for the optimizer and gains the most

recommend -> recommended

two-body is repeated